pkg/api/handlers/libpod/generate.go

```diff
@@ -105,8 +103,19 @@ func GenerateKube(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Read the default kubeGenerateType from containers.conf it the user doesn't specify it
+	generateType := query.Type
+	if generateType == "" {
+		config, err := runtime.GetConfig()
```

Comment:
Please use `GetConfigNoCopy` which is significantly faster as the config doesn't need to be marshaled. Since we're only reading, we don't need a copy.